docs/user_guide.md

```diff
@@ -248,3 +249,75 @@ If you are using a self-signed cert, make sure to copy the CA cert into ```/etc/
 
 When an image is signed, it has a tick shown in UI; otherwise, a cross sign(X) is displayed instead.  
 ![browse project](img/content_trust.png)
+
+### Vulnerability scan via Clair 
+**CAUTION: Clair is an optional installing component, please make sure you have already enable it in your Harbor before you go through this part!**
+
+Static analysis of vulnerabilities is provided through open source project [Clair](https://github.com/coreos/clair). You can trigger scan action on any images you want to as well as scan all the images in Harbor at one time. Additionally, you can also set a policy to scan all the images at a specified time everyday.
+
+**Vulnerability metadata** 
+
+Clair depends on the vulnerability metadata to complete the analysis process. After the first initial installation, Clair will automatically start to update the metadata database from different vulnerability repositories. The updating process might be take a while based on the data size and network situations etc.. If the database is not fully ready, there will be a warning tips at the footer of the repositories datagrid view.
+![browse project](img/clair_not_ready.png)
+
+The 'database not fully ready' warning message is also displayed in the **'Vulnerability'** tab of **'Configuration'** section under **'Administration'** for your awareness.
+![browse project](img/clair_not_ready2.png)
+
+Once the database is ready, an overall database updated timestamp will be shown in the **'Vulnerability'** tab of **'Configuration'** section under **'Administration'**. Click the overall timestamp above, the detailed timestamps list for different namespaces are dropped down.

```

Comment:
Click the overall timestamp above, the detailed timestamps list for different namespaces are dropped down.
==>
Click on the timestamp drop-down list, the timestamps of different namespaces are listed.